src/main/java/racingcar/Game.java

```diff
@@ -0,0 +1,67 @@
+package racingcar;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Game {
+    private List<String> cars;
+    private int times;
+    private List<Car> carList = new ArrayList<>();
+    private StringBuilder winner = new StringBuilder();
+
+    public Game(List<String> cars, int times) {
+        this.cars = cars;
+        this.times = times;
+    }
+
+    public List<Car> createCarList() {
+        if (cars.size() != 0) {
+            for (String carName : cars) {
+                Car newCar = new Car(carName);
+                carList.add(newCar);
+            }
+        }
+        return carList;
+    }
+
+    public void compareRandomNumber() {
+        for (Car car : carList) {
+            int GO_NUMBER = 4;
+            NumberGenerator numberGenerator = new NumberGenerator();
+            int randomNumber = numberGenerator.createRandomNumber();
+            if (GO_NUMBER <= randomNumber) {
+                car.setRacingSign();
+            }
+        }
+        OutputView outputView = new OutputView(carList);
+        outputView.printRacing();
+        System.out.println();
+    }
+
+    public int findMaximumRacingSign() {
+        int maximumRacingSign = 0;
+        for (Car car : carList) {
+            int racingSignLength = car.getRacingSign().length();
+            if (maximumRacingSign < racingSignLength) {
+                maximumRacingSign = racingSignLength;
+            }
+        }
+        return maximumRacingSign;
+    }
+    public void findWinner(int maximumRacingSign) {
+        OutputView outputView = new OutputView();
+        for (Car car : carList) {
+            if (car.getRacingSign().length() == maximumRacingSign) {
+                outputView.printWinner(car.getName());
```

Comment:
```suggestion
            if (car.isMaxPosition(maximumRacingSign)) {
                outputView.printWinner(car.getName());
```

객체지향적으로 하면 이렇게 됩니다 ㅎ